src/desktop/notification/notification.js

```diff
@@ -1,27 +1,30 @@
-import React,{Component} from 'react';
+import React, { Component } from 'react';
 import './notification.css';
 import Status from './statusicons/status';
-import {theme_color_panels,notify_status} from '../../globalvariables';
-import { FaExpand,FaBell, FaGithub, FaLinkedin, FaEnvelope, FaAt, FaStar } from 'react-icons/fa';
+import { theme_color_panels, notify_status } from '../../globalvariables';
+import { FaExpand, FaBell, FaGithub, FaLinkedin, FaEnvelope, FaAt, FaStar } from 'react-icons/fa';
 
-export default class Panel extends Component
-{
-    constructor(props)
-    {
+export default class Panel extends Component {
+    constructor(props) {
         super(props);
-        this.expand=this.expand.bind(this);
-        this.state={panelcolor:theme_color_panels,paneltype:'solid'};
+        this.expand = this.expand.bind(this);
+        this.state = { panelcolor: theme_color_panels, paneltype: 'solid' };
     }
-    expand()
-    {
-        this.props.parent_class.setState({ isFull: true });
+
+    //Fullscreen enter and exit code by aritroCoder
+    expand() {
+        if(this.props.parent_class.isFull=== true)
+            this.props.parent_class.setState({ isFull: false });
+        else{
+            this.props.parent_class.setState({ isFull: true });
+            document.webkitExitFullscreen();
```

Comment:
Why is there a webkit specific function? What about chrome?

Reply:
I have tested it in chrome its working there

Reply:
Its working in chrome, edge, opera , brave so I think it is no problem

Reply:
Actually can you tell me what document.webkitExitFullScreen is doing? IIRC The isFull flag will signal FullScreen React component to do that for you. Can you remove that and test if it works? Thanks!

Reply:
I had tried what you had told before but it was not working for me. The screen was getting into fullscreen mode but it was not reverting back. I have checked the official docs of react-fullscreen myself but I did not found any way of reverting back from fullscreen. isFull state seems to be working fine only for getting into fullscreen. So I used the DOM Document object to manipulate the fullscreen state. I updated the props just to make sure  it can get back into fullscreen again. I think this is the only way to solve this issue while having minimal impact on the existing codebase.